### PR TITLE
Feat check missing function

### PR DESCRIPTION
**Pull request by lachrist**
This will help prevent regression of: https://github.com/getappmap/appmap-agent-js/issues/219

### CODE REVIEW

File: packages/validate/lib/index.js [hunk 1]
```diff
@@ -190,6 +168,25 @@ const updateEventArray = (events, updates) => {
   );
 };
 
+const normalizeVersion = (version) => {
+  if (/^[0-9]$/.test(version)) {
+    return `${version}.0.0`;
+  } else if (/^[0-9]+\.[0-9]$/.test(version)) {
+    return `${version}.0`;
+  } else if (/^[0-9]+\.[0-9]+\.[0-9]+$/.test(version)) {
```
Comment (dustinbyrne) [inline packages/validate/lib/index.js:176]: The first regex doesn't allow multiple digits as in the major version.
The second regex doesn't allow multiple digits in the minor version.

File: packages/validate/lib/index.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ const versions = new Map([
 
 const keys = Array.from(versions.keys());
 
-const makeDesignator = JSON.stringify;
+const digest = JSON.stringify;
```
Comment (dustinbyrne) [inline packages/validate/lib/index.js:32]: This is nit-picky, but why not indicate that this is actually `JSON.stringify` and not some other implementation? Anybody will be familiar with `JSON.stringify` and not `digest`. The alias makes this a little harder to read.

File: packages/validate/lib/index.js [hunk 3]
```diff
@@ -83,29 +83,15 @@ const checkSchema = (data, options) => {
...
-      //   entity
-      // );
-      const designator = makeDesignator([
-        parent.name,
-        path2,
-        parseInt(lineno),
-        entity.static,
-        entity.name,
-      ]);
+      const designator = digest([entity.name, path2, parseInt(lineno), entity.static]);
```
Comment (dustinbyrne) [inline packages/validate/lib/index.js:94]: Is there a `path1`? :)

**Review by dustinbyrne — APPROVED** [review]

File: packages/validate/lib/index.js (hunk 3 above)
Comment (lachrist) [reply packages/validate/lib/index.js]: rebased

File: packages/validate/lib/index.js (hunk 2 above)
Comment (lachrist) [reply packages/validate/lib/index.js]: 1bf8410d38e289fd8de9d043aa10622f7fa847c6

File: packages/validate/lib/index.js (hunk 1 above)
Comment (lachrist) [reply packages/validate/lib/index.js:174]: rebased

**Pull request merged**